src/components/CommentList/index.js

```diff
@@ -0,0 +1,3 @@
+import CommentList from './CommentList';
+
+export default CommentList;
```

Comment:
In `index.js` it is better to use * selector.
Don't forget to destructure needed component where it is imported.
```suggestion
export * from './CommentList';
```